app/src/main/java/uk/ac/bath/petmatch/Database/DbHelper.java

```diff
@@ -101,9 +101,19 @@ public void generateSampleData() {
         user = new User("Dummy user", "user@petmatch.com", password, null);
         shelterUser = new User("Shelter user", "shelter@petmatch.com", password, dummyBath);
 
-        Pet goodBoy, mcGonagall;
+        Pet goodBoy, roz, artem, jan, josh, mcGonagall, hagrid, malfoy, dumbledore, snape;
+        // cats
         goodBoy = new Pet("Good boy", "He's the goodest boy", dummyBath, labrador);
+        roz = new Pet("Roz", "Is good at herding cows", bath, husky);
+        artem = new Pet("Artem", "Needs exercising often", dummyBath, beagle);
+        jan = new Pet("Jan", "Only eats pedigree food", bath, airedale);
+        josh = new Pet("Josh", "Likes it when you play soundtracks", dummyBath, chihuahua);
+        // dogs
         mcGonagall = new Pet("Minerva McGonagall", "Have a biscuit, Potter.", bath, tabby);
+        hagrid = new Pet("Hagrid", "Very hairy", dummyBath, ragdoll);
+        malfoy = new Pet("Draco Malfoy", "Weaselike in appearance", bath, siamese);
+        dumbledore = new Pet("Albus Percival Wulfric Brian Dumbledore", "Don't pity the dead, pity the living", dummyBath, britishshorthair);
```

Comment:
:heart_eyes:

Reply:
🐱 🐶

Reply:
@Jabzilla can you then please use this picture:

![Dumblecat](https://i.pinimg.com/originals/5e/d0/20/5ed020a3dadfeaa9f1422d5a9dc98bee.jpg)